x-pack/plugins/lens/public/visualizations/metric/visualization.tsx

```diff
@@ -376,7 +376,7 @@ export const getMetricVisualization = ({
 
   initialize(addNewLayer, state, mainPalette) {
     return (
-      state ?? {
+      state || {
```

Comment:
Why this change?

Reply:
I think I just accidentally changed as in a previous commit I changed the whole line, but restoring it I've rewritten this way. I'll change it back to the original version

Reply:
```suggestion
      state ?? {
```